app/utils/callback/lava.py

```diff
@@ -615,16 +677,25 @@ def add_tests(job_data, job_meta, lab_name, db_options,
         _store_lava_json(job_data, meta)
         groups = []
         cases = []
+        start_log_line = 0
+        log = yaml.load(job_data["log"], Loader=yaml.CLoader)
+        end_lines_map = {}
 
         for suite_name, results in job_data["results"].iteritems():
             if suite_name == "lava":
                 _add_login_case(meta, results, cases, 'auto-login-action')
+                start_log_line = _get_log_line_number(log,
```

Comment:
Maybe a function argument for the default line number rather than `or 0` which can be easily missed when reading the code.

Reply:
yes, it's rather ugly. I've seen it several times in the kernelci-backend code and followed the (anti)pattern. I'll fix it.